### ISSUE TITLE
Dependency to lru-cache should be updated

### ISSUE THREAD

**Issue by cnouguier** [calvinmetcalf/shapefile-js#173]
If we install shapefile-js we have the following error: `the Module not found: Error: Can't resolve 'yallist' in 'D:\PROJECTS\KDK\kano\node_modules\lru-cache'`. It seems the issue has been fixed ihttps://github.com/isaacs/node-lru-cache/issues/162.

**Comment by 3zzy** [comment]
Yes please, time to update deps again @calvinmetcalf Thanks!  :)
